app/javascript/utils/form/final_form/SearchField.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Field } from 'react-final-form'
+import classNames from 'classnames'
+import { Label, FieldError, HelpText } from './Field'
+
+export const SearchField = ({
```

Comment:
can you add explicit proptypes for this?

Seems like fields are our most complicated components so any extra documentation helps